kalite/distributed/templates/distributed/search_page.html

```diff
@@ -30,9 +30,6 @@ <h2 class="errorlist">{{ query_error }}</h2>
                             <a href="{% url 'learn' %}{{ node.path }}" class="{{ node.kind }}-{% if not node.available %}un{% endif %}available">{% trans node.title %}</a>
                         </li>
                     {% endfor %}
-                    {% if hit_max|get_item:category %}
```

Comment:
Querying by each type at a time is not very efficient, and also doesn't scale beyond Topic, Video, Exercise.